proposals/lip-0045.md

```diff
@@ -935,41 +804,27 @@ createTerminatedStateAccount(chainID, stateRoot = None):
         }
 
     create an entry in the terminated state substore with
-        storeKey = uint32be(chainID)
-        storeValue = serialization of terminatedState
-
-    return True
+        storeKey = chainID
+        storeValue = encode(terminatedStateSchema, terminatedState)
 ```
 
 #### createTerminatedOutboxAccount
 
 The `createTerminatedOutboxAccount` function creates an entry in the terminated outbox substore.
 
-##### Parameters
-
-* `chainID`: The ID of the chain.
-* `outboxRoot`: The outbox root to be stored in the terminated outbox account.
-* `outboxSize`: The outbox size to be stored in the terminated outbox account.
-* `partnerChainInboxSize`: The partner chain inbox size to be stored in the terminated outbox account.
-
-##### Returns
-
-A boolean indicating the success of the function execution.
-
-##### Execution
-
 ```python
-createTerminatedOutboxAccount(chainID, outboxRoot, outboxSize, partnerChainInboxSize):
+def createTerminatedOutboxAccount(chainID: ChainID, 
```

Comment:
As far as I see, this command is only called from one place right now, the "Message Recovery Initialization Command". This command also does not emit any event. Could it be useful to have an associated event for this command or do you think this is not necessary?

Reply:
Same as above, here I was thinking that the events emitted when processing that command would be enough, but we could add a new event for this

Reply:
There is not event emitted by the Message Recovery Initialization Command so you mean just the default event that says the command was executed successfully. This would not provide any information about the state change, but I guess the interoperability module state after the execution would contain the necessary information.

Reply:
I added two new events for the creation of a terminated state and outbox account respectively.

Reply:
Looks good now, I resolve this comment.